### PR TITLE
Update Bug triage guidelines with the current labels, add doc labels

### LINKED ISSUE #4166: Explain new labels in godot-docs
Add the new labels to bug_triage_guidelines.rst

### PR DESCRIPTION
**Pull request by Calinou**
This closes #4166.

### CODE REVIEW

File: community/contributing/bug_triage_guidelines.rst [hunk 1]
```diff
@@ -114,17 +142,45 @@ If one of the platform labels is used, it is then exclusive and the
 previous assumption doesn't stand anymore (so if it's a bug on e.g.
 Android and Linux exclusively, select those two platforms).
 
+Documentation labels
+~~~~~~~~~~~~~~~~~~~~
+
+In the `documentation repository <https://github.com/godotengine/godot-docs`__, we
+use the following labels:
+
+-  *Bug*: Incorrect information into an existing page. Not to be used for
```
Comment (clayjohn) [inline community/contributing/bug_triage_guidelines.rst:151]: Into -> in

Comment (Calinou) [reply community/contributing/bug_triage_guidelines.rst:151]: Good catch, fixed.

**Comment by NathanLovato** [comment]
Thanks as always, looking good. There's just a build error on the CI, could you check it out? I'll merge as soon as the CI passes.

**Comment by NathanLovato** [comment]
Thanks!

**Pull request merged**